src/core/server/http/integration_tests/router.test.ts

```diff
@@ -1839,3 +1838,49 @@ describe('ETag', () => {
       .expect(304, '');
   });
 });
+
+describe('Timeouts', () => {
+  let httpServer: HttpServer;
+  let router: Router;
+
+  beforeEach(() => {
+    httpServer = new HttpServer(logger, 'foo');
+    const enhanceWithContext = (fn: (...args: any[]) => any) => fn.bind(null, {});
+    router = new Router('', logger.get(), enhanceWithContext);
```

Comment:
This was in the `BasePathServer` test file, even if just directly testing core's `HttpServer`, so I moved it there. All other similar tests from the basePath server test file were already covered.

Reply:
Ah thanks, I forgot the http_server tests are 'not' IT atm, couldn't find this file